### PR TITLE
minor documentation fixes

### PR DESCRIPTION
**Pull request by rodney-onica**

### CODE REVIEW

File: docs/source/iot_hello_world/iot_hello_world.rst [hunk 1]
```diff
@@ -21,6 +21,12 @@ Configure AWS IoT, and Download Certificates
     :alt: ../img/AWS_Management_Console_IOT.png
     :width: 1024px
 
+- If you are shown the AWS IoT splash screen, click **Get started** to proceed to the Console
+.. image:: ../img/hello-world-cert0.png
```
Comment (rodney-onica) [inline docs/source/iot_hello_world/iot_hello_world.rst]: The image is missing a pink `1` though perhaps it's obvious enough already.

**Comment by onica-emiller** [comment]
the intent, like with the other dev board components, is that its a hardwired component (like the onboard LED, etc) that a user can purpose however they like in their projects.  So giving it a specific label might discourage that kind of open use: "I wonder if its ok to use the "safe mode boot" button for other things?" . Not naming it leaves it open for users to hack w it as they please.

The safe mode implementation of the button is covered in the docs if that helps:
https://docs.onica.com/projects/iotanium/en/preview/features/features.html#safe-mode-boot

Do you think "multi purpose button" is closer to what you are looking for?  We could do that if you like?

**Comment by rodney-onica** [comment]
Then "Status LED" should just be "LED", given that it can be used by an application for any purpose?

"Multi-purpose" would be better, then the implication would be that "safe mode" isn't the _only_ thing it's for. 

I don't know that users would consider using that button in an application:  buttons and indicators on a dev board itself are (in my limited experience) for the operation of the board (e.g. a reset button, or power-on indicator).  Inputs and outputs for the application would be attached to IO pins, so that they are external to the board.

If this blue button were used by an application, then... the IoTanium could never be put in a box?

Also, if the application is listening for the button, does that disable safe mode booting?

**Comment by MrSurly** [comment]
> If this blue button were used by an application, then... the IoTanium could never be put in a box?

The blue thing is a switch.  A switch button would be the thing you put on top of it that would extend the switch operation outside of an enclosure.

**Pull request merged**